discord/utils.py

```diff
@@ -436,6 +436,17 @@ def resolve_invite(invite):
             return m.group(1)
     return invite
 
+def resolve_template(code):
+    from .template import Template # circular import
+    if isinstance(code, (Template, Object)):
+        return template.id
+    else:
+        rx = r'(?:https?\:\/\/)?discord(?:\.new|(?:app)?\.com\/templates)\/(.+)'
```

Comment:
This regex seems wrong. `discordapp.com/templates/...` doesn't seem to actually be valid (nor discord.com/templates/...).

```suggestion
        rx = r'(?:https?\:\/\/)?discord\.new\/(.+)'
```

Reply:
It's "template" not "templates"
![image](https://user-images.githubusercontent.com/34678047/83334883-2e20c780-a2b2-11ea-96a1-9387de24cd8c.png)